prawns/staking/docker-compose.yml

```diff
@@ -50,6 +51,7 @@ services:
       target: polkadotProcessor
     environment:
       DB_HOST: db
+      NETWORK: polkadot
```

Comment:
this needs to be dynamic

Reply:
If we keep it like this then I think our existing deployment code will continue to work?

Reply:
Or we could just commit to the one prawn per network and update the deployments / the graph